src/jobs/retrigger.js

```diff
@@ -17,6 +18,23 @@ const FULL_GRAPH_STATES = new Set([
 // We use public only operations on the queue here...
 const queue = new taskcluster.Queue();
 
+function replaceAllInstances(tasks, map) {
```

Comment:
The name of the function is a bit confusing. It's too generic (all instances) while what it does is replacing task IDs. Maybe something like recursiveUpdateTaskIds()?